web/components/liquidations.tsx

```diff
@@ -493,7 +505,7 @@ export const Liquidations: React.FC = () => {
                       <thead className="bg-gray-50 dark:bg-zinc-900 dark:bg-opacity-80">
                         <tr>
                           <th className="px-6 py-3 text-xs font-medium tracking-wider text-left text-gray-500 uppercase dark:text-zinc-400">
-                            Reward ID
+                            Reward IDS
```

Comment:
nitpick but now it looks like an abbrevation of something, the `s` should be lowercase 😅